src/main.rs

```diff
@@ -17,6 +17,8 @@ mod wowinterface_api;
 
 use crate::error::ClientError;
 
+pub const VERSION: &str = "0.2.1"; //env!("CARGO_PKG_VERSION");
```

Comment:
For testing. Remove before merging.

Reply:
This was great for testing. Made it easier.
You can remove this now in a commit.